docker-compose.yml

```
@@ -0,0 +1,79 @@
+      # test - 헬스체크에 사용할 명령어
+      #  - CMD-SHELL : /bin/sh -c를 통해 shell에서 명령어를 실행 (환경 변수 치환을 위함)
+      #  - pg_isready : PostgreSQL이 클라이언트 연결을 수락할 준비가 되었는지 확인
+      #  - -U : 연결을 시도할 사용자를 지정
+      #  - -d : 연결을 시도할 데이터베이스를 지정
+      #  -> 명령어가 exit code 0을 반환하면 healthy, 그 외에는 unhealthy로 판단
+      # interval - 헬스체크 실행 간격
+      # timeout - 각 헬스체크의 응답 대기 시간 (5초 이내에 응답해야 함)
+      # retries - unhealthy로 판정하기까지 연속 실패 횟수
+      # start_period - 컨테이너 시작 후 헬스체크 싪패해도 unhealthy로 판정하지 않는 유예 시간
+      # start_interval - start_period 동안의 헬스체크 간격
+      test: [ "CMD-SHELL", "pg_isready -U ${POSTGRES_USER} -d ${POSTGRES_DB}" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)